packages/eslint-config-airbnb-base/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "eslint-config-airbnb-base",
+  "version": "0.0.0",
```

Comment:
ah good call. i intend to tag this commit, so i'll update the version number too.